EIPS/eip-6997.md

```diff
@@ -0,0 +1,371 @@
+---
+eip: 6997
+title: ERC-721 with transaction validation step.
+description: A new validation step for transfer and approve calls, achieving a security step in case of stolen wallet.
+author: eduardfina (@eduardfina), Eduard López i Fina
+discussions-to: https://ethereum-magicians.org/t/erc721-with-a-validation-step/14071
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-05-07
+requires: 721
+---
+
+## Abstract
+
+This standard is an extension of [ERC-721](./eip-721.md). It defines a new validation functionality to avoid wallet draining, every `transfer` or `approve` will be locked waiting for validation.
```

Comment:
```suggestion
This standard is an extension of [ERC-721](./eip-721.md). It defines new validation functionality to avoid wallet draining: every `transfer` or `approve` will be locked waiting for validation.
```